examples/graphql-yoga/src/modules/starwars/starwars.ts

```diff
@@ -119,5 +102,5 @@ export const StarWars = createModule({
         }));
       },
     },
-  },
+  } as Resolvers,
```

Comment:
@cowboyd @wKich This current setup just makes typescript not complain but it doesn't help people writing the resolvers. Is there a better way to type this so we can actually use TypeScript for guidance in parameters?

Reply:
Yeah, you are right. It's better to declare resolvers:

```typescript
const resolvers: Resolvers = { /* ... */ }
```

Reply:
Another option is to wrap `createModule()`  to require the type of `resolvers` to be `Resolvers`. Another pattern I've become fond of is to use a function:

```ts
function moduleResolvers(): Resolvers {
}
```

and then invoke it like:

```ts
createModule({
  id: "starwars",
  dirname: import.meta.url,
  resolvers: moduleResolvers(),
});
```

Or as something that can be spread in:


```ts
function moduleResolvers(): { resolvers: Resolvers } {
 //....
}
```

```ts
createModule({
  id: "starwars",
  dirname: import.meta.url,
  ...moduleResolvers(),
});
```